backend/leanstore/concurrency-recovery/CRMG.hpp

```diff
@@ -51,6 +51,7 @@ class CRManager
    ~CRManager();
    // -------------------------------------------------------------------------------------
    void registerMeAsSpecialWorker();
+   void deleteSpecialWorker();
```

Comment:
Why do you need this?

Reply:
The memory for the special worker is not released. It does not pass address sanitizer.